### ISSUE TITLE
FAQ Page improvements + broken link

### ISSUE THREAD

**Issue by MathiasKandelborg** [discordjs/discord.js#5435]
Thank you for taking the time to read this issue. I appreciate your work!

The link to the official guide is faulty on this page: https://discord.js.org/#/docs/main/stable/general/faq

[It tries to redirect here](https://discordjs.guide/popular-topics/common-questions.html) which is just a 404 page.

Also, you might want to use the `target="_blank"` HTML attribute for these external links, to not break a chain of thought, and help people stay exactly where they want to be in the docs. Especially the discord server link that threw me off guard :sweat_smile: 

**Further details:**

- Priority: low, this is purely QoL updates and a single faulty link

Let me know if you want a PR, I'll be happy to take a look at these 'issues'.
I just need to know if you actually want to use the target blank feature, and where this official guide is located.

I hope you're having a nice day! :hugs:

**Comment by almostSouji** [comment]
Fixed in https://github.com/discordjs/discord.js/pull/5176

**Comment by MathiasKandelborg** [comment]
Well, it's not fixed, and that PR was made at the very beginning of the year, and is set for review on Feb. 11. 

Maybe notch the people, because even though this is a low priority issue... It shouldn't take 3 months to update a link, nor 40+ days to review this single change.

**Comment by almostSouji** [comment]
The issue will automatically be closed, once the actual fix is merged and arrives on stable - please keep it open until then, as you said, the issue is not yet fixed.

**Comment by MathiasKandelborg** [comment]
I apologize, I'm used to issues like these not being closed. I have no idea how it works around here, so I'm just glad it's going to be closed when the issue is actually resolved, which I hope is soon!
